src/platforms/android/androidauthenticationlistener.cpp

```diff
@@ -31,5 +35,21 @@ void AndroidAuthenticationListener::start(MozillaVPN* vpn, QUrl& url,
 
   url.setQuery(query);
 
-  AndroidUtils::instance()->startAuthentication(this, url);
+  QAndroidJniObject activity = QtAndroid::androidActivity();
+  jboolean supported = QAndroidJniObject::callStaticMethod<jboolean>(
+      "com/mozilla/vpn/PackageManagerHelper", "isWebViewSupported",
+      "(Landroid/content/Context;)Z", activity.object());
+  if (supported) {
+    AndroidUtils::instance()->startAuthentication(this, url);
+    return;
+  }
+  m_legacyAuth = new DesktopAuthenticationListener(this);
```

Comment:
Well, this doesn't fix the issue, right? We are just moving the problem elsewhere.
Of course, if the user has another default browser, the operation _could_ work.

Reply:
Indeed, if the default browser is too old this wont work. But the user (and support) can help themselves here as they can just switch default browsers.
- Note the old client does not have any checks of this sort, it's just uses the installed firefox( if available ) or google chrome.